src/build.d

```diff
@@ -347,6 +353,7 @@ auto predefinedTargets(string[] targets)
 {
     import std.functional : toDelegate;
     Appender!(void delegate()[]) newTargets;
+LtargetsLoop:
```

Comment:
Yuck!  Is there some other way?

Reply:
Yeah I could a `bool foundMatch = false` ... `foundmatch=true; break` ... `if (!foundMatch) { throw error`....

but it seems alot more complex than just saying, `break LtargetsLoop`.  Note that labeling loops is alot less complex than general labels and gotos.